modules/administration/pages/openscap.adoc

```diff
@@ -37,23 +45,22 @@ For more information about the {suse} Security Team, see  https://www.suse.com/s
 == Prepare Clients for an SCAP Scan
```

Comment:
Already in the intro, you want the user to install a package: spacewalk-oscap.  I think this should be mentioned here.  And here we'd better also say clearly that the packages are to be installed on the client.

Reply:
I went the other way, and took the line out of the intro. It doesn't really fit well there, I think.